pixelgl/gltriangles.go

```diff
@@ -75,7 +75,6 @@ func (gt *GLTriangles) SetLen(len int) {
 	if len < gt.Len() {
 		gt.data = gt.data[:len*gt.vs.Stride()]
 	}
-	gt.submitData()
```

Comment:
What is the performance impact of this? Submitting data after `SetLen` is actually necessary, because the size of the underlying VBO must change.